c/parallel/test/test_merge_sort.cpp

```diff
@@ -254,36 +254,6 @@ TEST_CASE("DeviceMergeSort::SortKeys works with input iterators", "[merge_sort]"
   REQUIRE(expected_keys == std::vector<TestType>(input_keys_ptr));
 }
 
-// TEST_CASE("DeviceMergeSort::SortKeys works with output iterators", "[merge_sort]")
```

Comment:
Two ideas:

1. Definitely add a comment to explain why the code is commented out instead of being removed entirely.
2. Maybe use `#ifdef NEVER_DEFINED` instead of comments, then at least you'll still get the syntax highlighting, and potentially other tooling still see this as C++ code.

Reply:
That's a good idea, I added them back